src/dapp-tests/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs }:
+
+let
+  ds-test-src = pkgs.fetchFromGitHub {
+    owner = "dapphub";
+    repo = "ds-test";
+    rev = "eb7148d43c1ca6f9890361e2e2378364af2430ba";
+    sha256 = "1phnqjkbcqg18mh62c8jq0v8fcwxs8yc4sa6dca4y8pq2k35938k";
+  } + "/src";
+
+  ds-test = pkgs.buildDappPackage {
+    src = ds-test-src;
+    name = "ds-test";
+    doCheck = false;
+  };
+
+in
+  pkgs.buildDappPackage {
+    name = "dapp-tests";
+    src = ./.;
+    deps = [ ds-test ];
+
+    solcFlags = "--evm-version constantinople";
```

Comment:
I think it's better to leave the default `evm-version` instead. The current version of solidity should default to Istanbul, methinks